src/utils/misc.ts

```diff
@@ -0,0 +1,3 @@
+export function sleep(ms = 0) {
```

Comment:
where is this used?

Reply:
https://github.com/itaifish/Sprocket-Pan/blob/misc-file-refactors/src/components/panels/script/ScriptPanel.tsx#L186

I think you meant to put it here but forgot?

Reply:
I'll update there!